### ISSUE TITLE
Does not work on Polymer 2.0

### ISSUE THREAD

**Issue by jawadsh123** [vguillou/swipe-action#9]
The component seems to not work on Polymer 2.0.
The reason that i could understand was the change of the tag <content> to <slot> and their behaviour.
Since I'm using this component in my project, I'm already working on a fix for this. Let me know if I should make a new file with the fix so that people with Polymer < 2.0 can still use the original component, or modify the same file.

**Comment by OvermindDL1** [comment]
I have significantly more development on my fork at https://github.com/OvermindDL1/swipe-action, could you check to see if it works with Polymer 2.0?  I've not tried 2.0 yet but I did make a lot of fixes and corrections since this repo seems dead (not accepting PR's and such, my fork is the open PR actually..).

**Comment by jawadsh123** [comment]
Alright, I'll go check out your repo and file an issue there if it doesn't work.

**Comment by jawadsh123** [comment]
I tried out your fork and it seems to not be working either. I've made a few changes to get it to work on 2.0.
Please allow issues on the fork so that we can continue there. 
I'd rather recommend that you create a new repo that can then be hosted on [webcomponents.org](https://www.webcomponents.org/) so that others can make use of it too. :)

**Comment by OvermindDL1** [comment]
@jawadsh123 Yeah I probably should since that may become the official'ish place for it since this one seems inactive, fully enabled.

I've not hosted anything on webcomponents.org yet, what are the requirements to do so, what features does the existing fork lack to allow that?

Feel free to PR, any chance you can make sure it works on Polymer 1.5+ as well (should have compatibility layers with Polymer 2.0 depending on the API's used if that is the version I think it is).

**Comment by OvermindDL1** [comment]
Oh also, you did try the bug-fixes branch yes?  Not master?  It was originally a fork repo for PR's so I use branches, master is identical to this repo.

I can merge the branch into master as well before you PR if you wish?

**Comment by vguillou** [comment]
Hello.
I'm very sorry for the lack of response lately. I must review my github settings, as i'm not receiving any notifications lately...
Anyway, I'm currently not using Polymer that much anymore, though I hope to come back to it in the future. But I do not have much time to look into this repo nowadays.

I'm open to transfering this rep to @OvermindDL1, if you are willing to maintain it of course.
Although you would have to host the demo on the lon run I suppose, and also the transfer may not work because you have a fork on your account.
@OvermindDL1 what do you think, should we try that ?

**Comment by OvermindDL1** [comment]
I can if you wish.  If you want to keep up with it I'd far prefer if you did as I have enough things to keep up with as it is.  ^.^
But I'd be up for that certainly.

I renamed my fork to `swipe-action-fork` so the `swipe-action` name is now clear on my profile.  I can host the demo as well if you want to transfer that repo over (is it a github pages repo?  If not I can make one for it then).

> I'm very sorry for the lack of response lately. I must review my github settings, as i'm not receiving any notifications lately...

All good, Github is a bit odd with those at times, I've had to finagle with my settings quite a bit.  ^.^;

**Comment by michael-learns** [comment]
Does this work now with Polymer 2.0?

**Comment by OvermindDL1** [comment]
> Does this work now with Polymer 2.0?

Supposedly with the PR by another my fork should work with 2.0.  :-)

**Comment by heruan** [comment]
Any update on this? Should I use the fork or was this one migrated to the new maintainer already?
